cameo/core/metabolite.py

```diff
@@ -64,14 +64,20 @@ def _relax_mass_balance_constrain(self, time_machine, absolute_bound):
             self.constraint.lb = -absolute_bound
             self.constraint.ub = absolute_bound
 
-    def knock_out(self, time_machine=None, absolute_bound=10000):
-        """'Knockout' a metabolite
+    def knock_out(self, time_machine=None, absolute_bound=10000, force_steady_state=False):
```

Comment:
I wouldn't go with that. Depending on how many reactions your metabolite is present sum(S.v) might become to be a very large number.